core/storage/voiceover/gae_models.py

```diff
@@ -60,7 +60,7 @@ class EntityVoiceoversModel(base_models.BaseModel):
     # A language-accent code, e.g., en-US.
     language_accent_code = datastore_services.StringProperty(
         required=True, indexed=True)
-    # A dict representing content IDs as keys and a nested dict as values.
+    # A dict representing content IDs as keys and nested dict as values.
```

Comment:
A dict containing content IDs as keys and nested dicts as values.

Reply:
Updated